modules/user_accounts/jsx/permissions.js

```diff
@@ -0,0 +1,337 @@
+/* global ReactDOM */
+
+var Role = React.createClass({
```

Comment:
Since we have `webpack` now, we can use modern `ES6` syntax and features. This way you could use `class` syntax and `imports` to have a component per file instead of all the components together in the same file. 
>Simple example in [dicom_archive](https://github.com/aces/Loris/blob/17.1-dev/modules/dicom_archive/jsx/dicom_archive.js)

It might be too much changes for this particular PR, but just a head-ups for the future